src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -443,81 +425,37 @@ public static bool RequiresWhiteSpace(Token tk)
         /// <returns>Minified Expression.</returns>
         public string GetMinifiedScript(string text, List<Token> tokens)
         {
-            Contracts.AssertValue(text);
-            Contracts.AssertValue(tokens);
-
-            var stringBuilder = new StringBuilder();
-
-            for (var i = 0; i < tokens.Count; i++)
-            {
-                var currentToken = tokens[i];
-                if (currentToken.Kind == TokKind.Comment)
-                {
-                    stringBuilder.Append(currentToken.Span.GetFragment(text));
-                }
-                else if (RequiresWhiteSpace(currentToken))
-                {
-                    stringBuilder.Append(" " + currentToken.Span.GetFragment(text) + " ");
-                }
-                else
-                {
-                    var tokenString = currentToken.Span.GetFragment(text);
-                    var shouldNotTrim = IsStringPartOfALargerInterpolatedString(i - 1 >= 0 ? tokens[i - 1] : null, currentToken, i + 1 < tokens.Count ? tokens[i + 1] : null);
-                    var newString = shouldNotTrim ? tokenString : tokenString.Trim();
-
-                    stringBuilder.Append(newString);
-                }
-            }
-
-            var result = stringBuilder.ToString();
-            return result;
+            return ScriptMinification.GetMinifiedScript(text, tokens);
         }
 
         /// <summary>
-        /// Determines if current token is string literal inside a larger interpolated string.
+        /// Given the expression and list of tokens that make up the expression, generates the minified version of the given expression.
         /// </summary>
-        /// <param name="precedingToken">A token before the current token.</param>
-        /// <param name="currentToken">Current token that might be a string literal inside a larger interpolated string.</param>
-        /// <param name="followingToken">A token after the current token.</param>
-        /// <returns>True if current token represents a string literal inside a larger interpolated string or false otherwise.</returns>
-        private static bool IsStringPartOfALargerInterpolatedString(Token precedingToken, Token currentToken, Token followingToken)
+        /// <param name="text">Expression to minify.</param>
+        /// <param name="options">Minification options.</param>
+        /// <returns>Minified Expression.</returns>
+        public string GetMinifiedScript(string text, MinificationOptions options = null)

```

Comment:
Do we need this if on 428 you are just calling the static method

Reply:
I need this method in the doc server side.